Website/src/components/Navbars/HomeNavbar.jsx

```diff
@@ -42,16 +22,15 @@ function HomeNavbar() {
           }}
         />
       ) : null}
-      <Navbar className={"fixed-top " + navbarColor} expand="lg" color="info">
+      <Navbar className={"fixed-top "} style={{backgroundColor: '#171b5d'}} expand="lg">
```

Comment:
Same previous comment. Please move all styles to the tkot.scss file under an appropriate CSS class name, then reference the CSS class name in the component.